### ISSUE TITLE
Clarify hacking.vector enums related to 3rd parties

### ISSUE THREAD

**Issue by whbaker** [vz-risk/veris#106]
As of now, one has several options for remote access/desktop/etc vectors. There's also a "partner connection or credential." I wonder how often "3rd party desktop" is confused with "a remote desktop connection used by a 3rd party" rather than an "off the shelf remote desktop solutions like LogMeIn." I also bet "partner" is often left out for remote connections to 3rd parties (in fact, I know it is). 

Bottom line, I think this could be cleaner. Maybe a set of different types of remote access solutions and then the same list for 3rd parties. I dunno. Worth a convo, though.

**Comment by gdbassett** [comment]
added some clarity to descriptions in [dev e3e4ff7]

**Comment by gdbassett** [comment]
Example:
3rd party:  "3rd party online desktop sharing (LogMeIn, Go2Assist) installed for general remote desktop use"
Desktop sharing: "Graphical desktop sharing (RDP, VNC, PCAnywhere, Citrix) installed for to support a documented use case"

**Comment by Spitler** [comment]
Would combining these two into a more general "Desktop Sharing Software" option make more sense? I am trying to understand what we gain from the separation and think it may be more confusing than enlightening.

**Comment by gdbassett** [comment]
Deferring to 1.4 so we can work out if we'll combine them or just change their descriptions

**Comment by gdbassett** [comment]
I think we could add "Desktop Sharing Software" in addition to the other two.  In updating to the version that has this, (likely a 1.3.2), we can mark "Desktop Sharing Software" true anywhere the other two are and update descriptions so that "Desktop Sharing Software" is chosen in the future.  Then at version "SplitEnum", the other two can become sub-enumerations of "Desktop Sharing Software"

**Comment by gdbassett** [comment]
Updated in [v1_3_2 3f4076c]

**Comment by gdbassett** [comment]
added to partner survey template 1.3.2 and removed the other two from the template (not the schema) to minimize confusion
